omron_os32c_driver/src/os32c_node.cpp

```diff
@@ -41,6 +43,8 @@ using eip::socket::TCPSocket;
 using eip::socket::UDPSocket;
 using namespace omron_os32c_driver;
 
+double RATE = 12.856;
```

Comment:
Parameterize the expected rate and permitted deviation, please.

Reply:
Ok, any suggestions for naming? 

- `~expected_frequency` (default=`12.856`)
- `~frequency_tolerance` (default=`0.1`)
- `~timestamp_min_acceptable` (default=`-1`)
- `~timestamp_max_acceptable` (default=`5`)